google-cloud-core/src/main/java/com/google/cloud/Policy.java

```diff
@@ -317,9 +428,13 @@ public boolean equals(Object obj) {
       return false;
     }
     Policy other = (Policy) obj;
-    return Objects.equals(bindings, other.getBindings())
-        && Objects.equals(etag, other.getEtag())
-        && Objects.equals(version, other.getVersion());
+    if (bindingsList.size() != other.getBindingsList().size()) {
+      return false;
+    }
+    for (int i = 0; i < bindingsList.size(); i++) {
+      bindingsList.get(i).equals(other.getBindingsList().get(i));
```

Comment:
sorry, missed this. Should this be returning something?

Reply:
oh, I missed this as well. Will fix.